test/test_roscpp/test/src/params.cpp

```diff
@@ -528,6 +528,38 @@ TEST(Params, mapBoolParam)
   ASSERT_TRUE(std::equal(map_b.begin(), map_b.end(), map_b2.begin()));
 }
 
+TEST(Params, paramTemplateFunction)
+{
+  EXPECT_TRUE( param::param<std::string>( "string", "" ) == "test" );
```

Comment:
This and the other EXPECT_TRUE on strings being equal should use EXPECT_EQ [docs on string comparisions](https://code.google.com/p/googletest/wiki/Primer#String_Comparison) It gives better visibility into the error reports.